src/main/java/bio/terra/cli/command/helperclasses/FormatFlag.java

```diff
@@ -0,0 +1,101 @@
+package bio.terra.cli.command.helperclasses;
+
+import bio.terra.cli.command.exception.SystemException;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectWriter;
+import java.io.PrintStream;
+import java.util.function.Consumer;
+import picocli.CommandLine;
+
+/**
+ * Command helper class that defines the --format flag and provides utility methods to printing a
+ * return value out in different formats.
```

Comment:
Worth mentioning this is meant to be used as a `@Mixin`?

Reply:
Added note to the class comment.